content/500-platform/550-platfrom-cli/commands.mdx

```diff
@@ -0,0 +1,229 @@
+| `--apikey`       | `string` | yes      | The API key `id`.<br/><br/>**Hint**: You can view your API key ids with the [`apikey show`](#apikey-show) command.            |
+
+### Prisma Accelerate
+
+### <inlinecode>platform</inlinecode>
+
+#### <inlinecode>accelerate enable</inlinecode>
+
+Enables Prisma Accelerate for the specified project.
+
+```bash
+npx prisma platform accelerate enable \
+--workspace cdlg1gzm000000000001b7m8b \
+--project ciiirb3z700017qyi0v7rcltd \
+--url "postgresql://username:password@host:port/database"
+--region us-east-1
+--apikey true
+--early-access
+```
+
+##### <span>Arguments</span>
+
+| Argument         | Type      | Required | Description                                                                                                                                                                                                                                                        |
+| ---------------- | --------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `--workspace -w` | `string`  | yes      | The workspace `id`.<br/><br/> **Hint:** You can view your workspace ids with the [`workspace show`](#workspace-show) command.                                                                                                                                      |
+| `--project` `-p` | `string`  | yes      | The project `id`.<br/><br/>**Hint**: You can view your project ids with the [`project show`](#project-show) command.                                                                                                                                               |
+| `--url`          | `string`  | yes      | Your database connection string.                                                                                                                                                                                                                                   |
+| `--region`       | `string`  | no       | The region for Accelerate’s managed connection pool.<br/><br/>View a list of available regions [here](/accelerate/faq#what-regions-is-accelerates-connection-pool-available-in).<br/><br/>**Hint**: Select the region _nearest_ your database for optimal latency. |
+| `--apikey`       | `boolean` | no       | If yes, a new API key will be generated for the associated project.                                                                                                                                                                                                |
+
+#### <inlinecode>accelerate disable</inlinecode>
+
+Disables Prisma Accelerate for the specified project.
+
+```bash
+prisma platform accelerate disable \
```

Comment:
```suggestion
npx prisma platform accelerate disable \
```
Just adding for consistency with the other commands

Reply:
Nice catch!